mobile-app/app/screens/AppNavigator/screens/Dex/hook/SlippageTolerance.ts

```diff
@@ -0,0 +1,30 @@
+import { SlippageTolerancePersistence } from '@api'
+import { useLogger } from '@shared-contexts/NativeLoggingProvider'
+import BigNumber from 'bignumber.js'
+import { useEffect, useState } from 'react'
+
+export interface SlippageTolerance {
```

Comment:
we are using this interface somewhere else? if not can remove `export`?

Reply:
This is the only interface that we have

Reply:
`export` can be removed, I don't see any other hook/component using it